docs/samples/git-setup/policytemplates/version_4.Y.Z/sno-ran-du/sno-ran-du-pg-v4-Y-Z-v4-full-DU.yaml

```diff
@@ -0,0 +1,376 @@
+apiVersion: policy.open-cluster-management.io/v1
```

Comment:
Is this DU profile based on the OCP 4.17? I wonder if adding a comment to mention that might be helpful.

Reply:
agree